### ISSUE TITLE
Midterm Peer Review

### ISSUE THREAD

**Issue by YitianSusanLin** [medhavi27/Hot-Spotify-Tracks#9]
This group wants to investigate the correlation between the features of the song and its popularity. 

Good Points:
1. I like the idea how you mimic the reality when you divide the training and test set by making all songs in training sets released earlier than those in test set. 
2. You tried multiple regressions on current data and compare their absolute mean error, which is very informative and gives insight about what to do for next step. 
3. You already realize there are fewer enough-popular songs than not-so-popular songs so the data may be skewed. Therefore, you can try more regularizer for this feature. 

Improvement:
1. I am sure there's some inaccuracy in the data like NAN. You can use the way we learnt in class to clean/preprocess the data first. 
2. For the histograms, it's better if you can add what does the x-axis and y-axis represent. 
3. I was wondering how you are going to find more features to try as mentioned in your "What's Next" section.
